config_samples/base/testnet/hoodi_vaults_testnet_config.json

```diff
@@ -0,0 +1,439 @@
+{
+  "unverified_contracts": {
+    "0x4d0E05677c1005C4C808cb5e824abE99b2b39497": "N/A",
+    "0x49335f718631a7aA201B459B499AFfdc1d8Ca774": "N/A",
+    "0xfF772cd178D04F0B4b1EFB730c5F2B9683B31611": "N/A",
```

Comment:
:warning:  It's an EOA, not a contract, see https://docs.lido.fi/deployed-contracts/hoodi-lidov3#%EF%B8%8F-special-accounts-and-addresses